server/mvcc/kvstore_txn.go

```diff
@@ -212,6 +214,12 @@ func (tw *storeTxnWrite) put(key, value []byte, leaseID lease.LeaseID) {
 	tw.changes = append(tw.changes, kv)
 	tw.trace.Step("store kv pair into bolt db")
 
+	// big value metrics
+	valueSize := len(value)
+	if valueSize > BigValueSize {
+		putBigValue.WithLabelValues(string(key)).Set(float64(valueSize))
```

Comment:
We might log Warn (above some threshold) that that there is big value (without speechifying key) and with trottling to not log too frequently. 

Or we need a 'compaction-refreshed' metric that shows distribution of value sizes.

Reply:
add warn log. But how frequently record one time.
I add a hot key metric, but it use to alert and define alert condition on alert module.

Reply:
I would consider wrapping this in standard throttler usage: https://pkg.go.dev/golang.org/x/time/rate#Limiter.Allow

Reply:
yes, it's better.